easybuild/easyconfigs/g/GDAL/GDAL-3.7.1-foss-2023a.eb

```
@@ -0,0 +1,82 @@
+easyblock = 'CMakeMake'
+
+name = 'GDAL'
+version = '3.7.1'
+
+homepage = 'https://www.gdal.org'
+description = """GDAL is a translator library for raster geospatial data formats that is released under an X/MIT style
+ Open Source license by the Open Source Geospatial Foundation. As a library, it presents a single abstract data model
+ to the calling application for all supported formats. It also comes with a variety of useful commandline utilities for
+ data translation and processing."""
+
+toolchain = {'name': 'foss', 'version': '2023a'}
+toolchainopts = {'usempi': True}
+
+source_urls = ['https://download.osgeo.org/%(namelower)s/%(version)s/']
+sources = [SOURCELOWER_TAR_XZ]
+patches = ['%(name)s-3.6.2_fix-python-CC-CXX.patch']
+
+checksums = [
+    {'gdal-3.7.1.tar.xz': '9297948f0a8ba9e6369cd50e87c7e2442eda95336b94d2b92ef1829d260b9a06'},
+    {'GDAL-3.6.2_fix-python-CC-CXX.patch': '859b874b0c8ff7626a76d51f008bf05b7f89a35b325bdd1d126d2364154acc63'},
+]
+
+builddependencies = [
+    ('CMake', '3.26.3'),
+    ('pkgconf', '1.9.5'),
+    ('Bison', '3.8.2'),
+]
+dependencies = [
+    ('Python', '3.11.3'),
+    ('netCDF', '4.9.2'),
+    ('expat', '2.5.0'),
+    ('GEOS', '3.12.0'),
+    ('SQLite', '3.42.0'),
+    ('libxml2', '2.11.4'),
```

I think `libarchive` is required to fix `libgdal.so.33: undefined symbol: archive_read_support_format_rar5`

```suggestion
    ('SQLite', '3.42.0'),
    ('libarchive', '3.6.2'),
    ('libxml2', '2.11.4'),
```